tests/stream/test_ffmpegmux.py

```diff
@@ -121,6 +121,48 @@ def run(self):
             ("ffmpegmux", "debug", " bar"),
         ]
 
+    def test_validate_timeout_default_value(self, session: Streamlink):
+        session.options.update({"ffmpeg-no-validation": False})
+
+        class MyFFmpegVersionOutput(FFmpegVersionOutput):
+            def run(self):
+                self.onstdout(0, "ffmpeg version 0.0.0 default")
+                return True
+
+        with (
+            patch("streamlink.stream.ffmpegmux.which", return_value="/usr/bin/ffmpeg"),
+            patch("streamlink.stream.ffmpegmux.FFmpegVersionOutput", side_effect=MyFFmpegVersionOutput) as mock_versionoutput,
+        ):
+            result = FFMPEGMuxer.command(session)
+
+        assert result == "/usr/bin/ffmpeg"
+        assert mock_versionoutput.call_args_list == [
+            call(["/usr/bin/ffmpeg", "-version"], timeout=4.0),
+        ]
+
+    def test_validate_timeout_custom_value(self, session: Streamlink):
+        session.options.update({
+            "ffmpeg-no-validation": False,
+            "ffmpeg-validation-timeout": 9.5,
+        })
+
+        class MyFFmpegVersionOutput(FFmpegVersionOutput):
+            def run(self):
+                self.onstdout(0, "ffmpeg version 0.0.0 custom")
+                return True
+
+        with (
+            patch("streamlink.stream.ffmpegmux.which", return_value="/usr/bin/ffmpeg"),
+            patch("streamlink.stream.ffmpegmux.FFmpegVersionOutput", side_effect=MyFFmpegVersionOutput) as mock_versionoutput,
+        ):
+            result = FFMPEGMuxer.command(session)
+
+        assert result == "/usr/bin/ffmpeg"
+        assert mock_versionoutput.call_args_list == [
+            call(["/usr/bin/ffmpeg", "-version"], timeout=9.5),
+        ]
```

Comment:
Instead of adding a second test case, different timeout values (default and custom) should be parametrized (`@pytest.mark.parametrize(...)`).

Btw, there's an additional blank line below that causes a linting error.